randobot/handler.py

```diff
@@ -161,13 +173,512 @@ async def race_data(self, data):
             await self.unpin_message(self.state['pinned_msg'])
             del self.state['pinned_msg']
 
+    @monitor_cmd
+    async def ex_s7(self, args, message):
+        """
+        Handle !s7 commands.
+
+        Set up room for Draft Mode.
+        """
+        if self._race_in_progress() or not self._is_s7_race():
```

Comment:
Think we should probably move the `self._is_s7_race()` condition further in so it only applies for tournament/qualifier. There's no need to check this for practice races, and will probably just cause confusion

Reply:
The idea was to only enable the use of these commands for draft races to not compromise the functionality of randobot outside of said races such as league races, other tournaments, etc.

Reply:
I think it's fine for the `!s7` command specifically since it's the entry point for the feature, can only be used by monitors anyway and I doubt someone would set it off accidentally. Probably will be less trouble for users if it's always available rather than restricted based on room name.